### DISCUSSION TITLE
graphQl filter with regex filter?

### DISCUSSION THREAD

**Discussion by TheOneAndOnlyGarcia** [directus/directus#3669] [Archive]
Hello, 

is it possible to apply a regex filter to a Directus graphql query? Like this?

```graphQl
{
  directus {
    items {
      something(filter: {title: {regex: "/([ A-Z])\w+"}}) {
        id
        title
      }
    }
  }
}
```

If this is not implemented, wouldn't that be a good feature?

**Comment by philleepflorence** [comment]
These are the currently supported filters: [Supported Operators](https://docs.directus.io/reference/filter-rules.html#supported-operators)

Since the filter operators have to translate to the DB vendor specific operators, I would check to see if it is supported by the all (or most) of the DBs, as well as: [Knex.js,](http://knexjs.org/#Builder-where) the query builder used by Directus.

What is the DB vendor you use?

**Reply by TheOneAndOnlyGarcia** [reply]
@philleepflorence i am using Postgres. thank you! :)

**Comment by rijkvanzanten** [comment]
> is it possible to apply a regex filter to a Directus graphql query? Like this?

No, there's an open ticket to implement regex support, see https://github.com/directus/directus/issues/2917 for more info 👍🏻
